src/main/java/ua/edu/internship/interview/service/business/UserSkillsService.java

```diff
@@ -0,0 +1,70 @@
+package ua.edu.internship.interview.service.business;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import ua.edu.internship.interview.data.documents.SkillDocument;
+import ua.edu.internship.interview.data.documents.UserSkillsDocument;
+import ua.edu.internship.interview.data.repository.SkillRepository;
+import ua.edu.internship.interview.data.repository.UserSkillRepository;
+import ua.edu.internship.interview.service.dto.user.skill.UserSkillsDto;
+import ua.edu.internship.interview.service.mapper.UserSkillsMapper;
+import ua.edu.internship.interview.service.utils.exceptions.InvalidInputException;
+import ua.edu.internship.interview.service.utils.exceptions.NoSuchEntityException;
+import java.util.List;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class UserSkillsService {
```

Comment:
Do I need to make a request to the User Service to check if user with specified ID exists, every time I create/update skills for user?

Reply:
To clarify: I meant that I need to make a request on User Service to validate, that user with specified ID exists.

Reply:
Will be resolved in next PR. Will add Feign Client Lib to make requests on user service.